core-tests/src/test/java/org/mule/runtime/core/construct/FlowProcessingStrategyTestCase.java

```diff
@@ -39,6 +37,9 @@
   @Mock
   private MuleConfiguration configuration;
 
+  @Mock
+  private SchedulerService schedulerService;
```

Comment:
This is an unused leftover from my ProcessingStrategyFactory changes which you then implemented. Removing.